core/events.go

```diff
@@ -35,6 +35,10 @@ type PendingLogsEvent struct {
 	Logs vm.Logs
 }
 
+// PendingStateEvent is posted pre mining and notifies of pending state changes.
+type PendingStateEvent struct {
+}
```

Comment:
Out for the weekend, no laptop. Will fix Sunday/Monday.
On Feb 26, 2016 7:34 PM, "Jeffrey Wilcke" <notifications@github.com> wrote:

> In core/events.go
> <https://github.com/ethereum/go-ethereum/pull/2267#discussion_r54274546>:
>
> > @@ -35,6 +35,10 @@ type PendingLogsEvent struct {
> >  	Logs vm.Logs
> >  }
> >
> > +// PendingStateEvent is posted pre mining and notifies of pending state changes.
> > +type PendingStateEvent struct {
> > +}
>
> Sorry but could you move this one line up making it one line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethereum/go-ethereum/pull/2267/files#r54274546>.
>